api/harborapi_test.go

```diff
@@ -77,19 +80,28 @@ func init() {
 
 }
 
-func request(_sling *sling.Sling, acceptHeader string, authInfo ...usrInfo) (int, []byte, error) {
+func request(_sling *sling.Sling, acceptHeader string, authInfo ...usrInfo) (int, int, []byte, error) {
```

Comment:
shouldn't directly return any id(int), better return HEAD structure and get the id in separate function.